.github/workflows/stale.yml

```diff
@@ -39,7 +42,6 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3.5.2
       - name: Remove 'stale' label
-        run: gh issue edit ${{ github.event.issue.number }} --remove-label $stale_label
-        env:
```

Comment:
You dropped the `env:`. I'll push a fix at the same time as tweaking the conditional..

Reply:
Thanks. This was not planned.

Reply:
Looks good. Let's try this.